go.mod

```diff
@@ -78,18 +79,18 @@ require (
 	github.com/prometheus/common v0.9.1
 	github.com/prometheus/tsdb v0.8.0 // indirect
 	github.com/rancher/dynamiclistener v0.2.1-0.20200418023342-52ede5ec9234
-	github.com/rancher/kontainer-engine v0.0.4-dev.0.20200512184531-c395feca5b3b
+	github.com/rancher/kontainer-engine v0.0.4-dev.0.20200327180925-6d88ca2af46f
 	github.com/rancher/machine v0.15.0-rancher25
-	github.com/rancher/norman v0.0.0-20200417060332-5c2a13443bb8
+	github.com/rancher/norman v0.0.0-20200326201949-eb806263e8ad
 	github.com/rancher/rdns-server v0.0.0-20180802070304-bf662911db6a
 	github.com/rancher/remotedialer v0.2.6-0.20200401224317-04cc22157b83
-	github.com/rancher/rke v1.1.1
+	github.com/rancher/rke v1.1.0-rc9.0.20200327175519-ecc629f2c3d5
```

Comment:
Need to update this after merging https://github.com/rancher/rke/pull/2060 with the commit from the release/v1.1 branch

Reply:
@deniseschannon should I get this tagged as a separate release for rke or reference the commit directly as `v1.1.1-rc...` once it is merged?

Reply:
Will also need to update vendor with changes from back port PR https://github.com/rancher/rke/pull/2060 related to original PR https://github.com/rancher/rke/pull/2044